### ISSUE TITLE
Tab Navigation content outputed as clickable to screen readers

### ISSUE THREAD

**Issue by nikolab-pn** [telekom/scale#957]
Hi,

looks like all texts inside the tabs are outputed as clickable to the screen reader tool (JAWS). This is confusing for screen reader users.

**Comment by acstll** [comment]
Would you be so kind and share some code snippet so we can test this?

Thanks in advance 🙏

**Comment by marvinLaubenstein** [comment]
Hi @nikolab-pn, sorry for the long wait.
I have checked the problem again. Unfortunately, I could not reproduce it. I tried with Windows Voice-Assistant, VoiceOver and JAWS on all major browsers. 
Is this still an issue on your side ?

**Comment by nikolab-pn** [comment]
Hi @marvinLaubenstein,

here is the video of the issue in Firefox 110.0 on macOS 13.2.1  using VoiceOver but it should be simmilar with other tools. "clickable" part is read out when I focus text part. I use VO key + arrow right to navigate to it when after tab panel is slected.

VO key  = Control + Option or Caps Lock

https://user-images.githubusercontent.com/98950408/221534102-6c7fad5d-52bd-44e7-82d8-3b75288823bd.mov

**Comment by marvinLaubenstein** [comment]
Hi @nikolab-pn, thanks again for the nice video. Now I see the issue. 
Based on your description, we had thought about a different placement of the issue, sorry for the missunderstanding. 

We will look at it again in detail and will fix it.
